league_rpc_linux/const.py

```diff
@@ -0,0 +1,45 @@
+# Discord Application: League of Linux
+DEFAULT_CLIENT_ID = "1185274747836174377"
+DISCORD_PROCESS_NAMES = ["Discord", "DiscordPTB", "DiscordCanary", "electron"]
+LEAGUE_OF_LEGENDS_LOGO = "https://github.com/Its-Haze/league-rpc-linux/blob/master/assets/leagueoflegends.png?raw=true"
+SMALL_TEXT = "github.com/Its-Haze/league-rpc-linux"
+CURRENT_PATCH = "13.24.1"
+
+
+ALL_GAME_DATA_URL = "https://127.0.0.1:2999/liveclientdata/allgamedata"
+
+BASE_SKIN_URL = "https://ddragon.leagueoflegends.com/cdn/img/champion/tiles/"
+BASE_CHAMPION_URL = (
+    f"http://ddragon.leagueoflegends.com/cdn/{CURRENT_PATCH}/img/champion/"
+)
+BASE_MAP_URL = f"http://ddragon.leagueoflegends.com/cdn/{CURRENT_PATCH}/img/map/map"
```

Comment:
If we upload the map image to "assets" folder. and point to the github url.. then we don't have to specify the current_patch. Since the map image should never change.

But i know u said you had found the map from community dragon so i will wait until you update the PR with that.

Reply:
Adjusted to cdragon